core/src/components/toast/toast-interface.ts

```diff
@@ -4,7 +4,6 @@ import type { IonicSafeString } from '../../utils/sanitization';
 export interface ToastOptions {
   header?: string;
   message?: string | IonicSafeString;
-  cssClass?: string | string[];
```

Comment:
I don't think this was supposed to be removed, `cssClass` still exists for Toast: https://github.com/ionic-team/ionic-framework/blob/FW-4923/core/src/components/toast/toast.tsx#L123

Reply:
Good catch! Thank you!!